src/core/dbus-service.c

```diff
@@ -718,6 +748,56 @@ static int bus_service_set_transient_property(
         if (streq(name, "ReloadSignal"))
                 return bus_set_transient_reload_signal(u, name, &s->reload_signal, message, flags, error);
 
+        if (streq(name, "ExtraFileDescriptor")) {
```

Comment:
I think we want plural here so `ExtraFileDescriptors`

Reply:
Ok! I just used what `OpenFile` did and use singular everywhere. Happy to use plural